doc/classes/CameraEffects.xml

```diff
@@ -1,31 +1,43 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="CameraEffects" inherits="Resource" version="4.0">
 	<brief_description>
+		Contains camera-specific effects such as depth of field and exposure override.
 	</brief_description>
 	<description>
+		Contains camera-specific effects such as depth of field and exposure override. These effects are split from [Environment] so that different cameras can use different values for depth-of-field and exposure, without having to duplicate the [Environment] resource.
```

Comment:
This sentence is a bit unwiedly, could someone help reformulate it while keeping the meaning?

Reply:
Is this better?
"Controls camera-specific effects (like exposure and DOF). CameraEffects override Environment effects, so they are useful for games that use multiple cameras in the same environment."

Reply:
DoF isn't defined in Environment anymore, so that wouldn't be correct.
In the worst case, we could drop the explanation but I tried to justify why DoF was moved from Environment to CameraEffects in a single sentence :slightly_smiling_face:

Reply:
I see. In that case, I think it would be a good idea to cut the explanation because users don't need to know that these effects were previously available in the Environment resource.

Reply:
I agree, just cut the reference to Environment entirely. We can mention this rationale when we update the post-processing effects tutorial.

Reply:
Done, I just left the "see also" note.